crates/fluvio-types/src/event.rs

```diff
@@ -40,6 +40,10 @@ impl SimpleEvent {
         listener.await
```

Comment:
bump up version

Reply:
Ah so I wanted to ask, what do you think of renaming this to `StickyEvent`? It's an event because you listen for it, but after it is triggered once it always yields immediately. I think it communicates the behavior better, what do you think?

Reply:
reasonable but we have to keep old name around and deprecated until next major version